packages/discord.js/src/client/websocket/WebSocketManager.js

```diff
@@ -119,11 +93,11 @@ class WebSocketManager extends EventEmitter {
   /**
    * Emits a debug message.
    * @param {string} message The debug message
-   * @param {?WebSocketShard} [shard] The shard that emitted this message, if any
+   * @param {?number} [shardId] The id of the shard that emitted this message, if any
    * @private
    */
-  debug(message, shard) {
-    this.client.emit(Events.Debug, `[WS => ${shard ? `Shard ${shard.id}` : 'Manager'}] ${message}`);
+  debug(message, shardId) {
+    this.client.emit(Events.Debug, `[WS => ${shardId ? `Shard ${shardId}` : 'Manager'}] ${message}`);
```

Comment:
```suggestion
    this.client.emit(Events.Debug, `[WS => ${typeof shardId === 'number' ? `Shard ${shardId}` : 'Manager'}] ${message}`);
```

Reply:
This is (just) a debug message, wouldn't it be correct to include wrongly passed stuff in the log here instead of only making it log if it's a number?